array_methods/task1.js

```diff
@@ -0,0 +1,19 @@
+const numbers = [2, -5, 0, 7, -3, 0, 10, -8]
+let positiveCount = 0;
+let negativeCount = 0;
+let zeroCount = 0;
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i]>0){
+        positiveCount++
+    }
+    if (numbers[i]<0){
+        negativeCount++
+    }
```

Comment:
either add `continue` or use `else if`

right now even if first condition is truthy you still check others